### PR TITLE
docs: translate /docs/hertz/overview/_index.md

### PR DESCRIPTION
**Pull request by Haswf**
Hi there,

I am an early adapter of Hertz back in 2021, it  is so great it has made its way to the open source community. I came across the overview of Hertz and found some improvement could be done on its English translation. It is my best effort to make it fluent to read while preserving the original meaning.

### CODE REVIEW

**Comment by Haswf** [comment]
@amoblin Would you like to review this translation for me?

**Comment by GuangmingLuo** [comment]
> @GuangmingLuo Would you like to review this translation for me?

well done and thanks for your contribution

File: content/en/docs/hertz/overview/_index.md [hunk 1]
```diff
@@ -7,45 +7,50 @@ description: >
 ---
 
 ## CloudWeGo-Hertz
-Hertz [həːts] is a high-usability, high-performance and high-extensibility Golang HTTP framework that helps developers build microservices.
-It was designed with reference to other open-source frameworks like [fasthttp](https://github.com/valyala/fasthttp)、[gin](https://github.com/gin-gonic/gin)、[echo](https://github.com/labstack/echo) and combined with the internal requirements in ByteDance.
-At present, it has been widely used inside ByteDance. Nowadays, more and more microservices use Golang. If you have requirements for microservice performance and hope that the framework can fully meet the internal customizable requirements, Hertz will be a good choice.
+Hertz [həːts] is a high-performance, extensible HTTP framework for Go. It's designed to simplify building microservices for developers.
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/overview/_index.md:10]: We would like to remain the three-high statement while others are well optimized.

File: content/en/docs/hertz/overview/_index.md [hunk 2]
```diff
@@ -7,45 +7,50 @@ description: >
...
 
+
+
 ## Related Projects
-- [Netpoll](https://github.com/cloudwego/netpoll): A high-performance network library. Hertz integrated by default.
-- [Hertz-Contrib](https://github.com/hertz-contrib): A partial extension library of Hertz, which users can integrate into Hertz through options according to their needs.
-- [Example](https://github.com/cloudwego/hertz-examples): Use examples of Hertz.
+- [Netpoll](https://github.com/cloudwego/netpoll): A high-performance network library. Hertz uses it by default.
+- [Hertz-Contrib](https://github.com/hertz-contrib): A collection of Hertz extensions.
+- [Example](https://github.com/cloudwego/hertz-examples): An example of a Hertz project.
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/overview/_index.md:53]: Actually, it is not just "an" example but multiple examples.

File: content/en/docs/hertz/overview/_index.md [hunk 3]
```diff
@@ -7,45 +7,50 @@ description: >
...
+ ![Performance](/img/docs/hertz_performance.png)
+Please refer to https://github.com/cloudwego/hertz-benchmark for more details about benchmarking.
 
-- High extensibility
+### High extensibility
 
-  Hertz adopts a layered design, providing more interfaces and default extension implementations. Users can also extend by themselves. At the same time, thanks to the layered design of the framework, the extensibility of the framework will be much greater. At present, only stable capabilities are open-sourced to the community. More planning refers to [RoadMap](https://github.com/cloudwego/hertz/blob/main/ROADMAP.md).
+Hertz adopts a layered architecture to deliver a "batteries included" experience. Due to its layered design, the framework is high extensible while its core functionality remains robust. Hertz comes with default implementations for many modules but also enables users to extend them to fit their own needs. At present, only stable capabilities have been made available to the open source community. More planning refers to [RoadMap](https://github.com/cloudwego/hertz/blob/main/ROADMAP.md).
```
Comment (GuangmingLuo) [inline content/en/docs/hertz/overview/_index.md:33]: highly extensible ?

Comment (Haswf) [reply content/en/docs/hertz/overview/_index.md]: @GuangmingLuo  Sorry for the delay. Would you like to take a look at changes I've made?

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**